package.json

```diff
@@ -20,12 +30,41 @@
   },
   "dependencies": {
     "babel-types": "^7.0.0-alpha.15",
-    "babylon": "^7.0.0-alpha.16",
+    "babylon": "^v7.0.0-beta.18",
     "chalk": "^2.0.1",
-    "commander": "^2.11.0"
+    "commander": "^2.11.0",
+    "ejs": "^2.5.7",
+    "purecss": "^1.0.0"
   },
   "devDependencies": {
-    "jest": "^20.0.4"
+    "babel-core": "6.25.0",
+    "babel-loader": "7.1.1",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-preset-env": "1.6.0",
+    "babili-webpack-plugin": "0.1.2",
+    "classnames": "2.2.5",
+    "css-loader": "0.28.4",
+    "exports-loader": "0.6.4",
+    "jest": "^20.0.4",
+    "mkdirp": "0.5.1",
+    "normalize.css": "7.0.0",
+    "postcss": "6.0.8",
+    "postcss-apply": "0.8.0",
+    "postcss-cssnext": "3.0.2",
+    "postcss-import": "10.0.0",
+    "postcss-import-url": "2.2.0",
+    "postcss-loader": "2.0.6",
+    "postcss-mixins": "6.0.1",
+    "postcss-nesting": "4.0.1",
```

Comment:
postcss here is an overkill .. remove all these plugins and simply use css-modules.